src/workloads/docs/RandomSampler.yml

```diff
@@ -13,12 +13,11 @@ Clients:
 Actors:
 - Name: Loader
   Type: Loader
-  Threads: 50
+  Threads: 1
```

Comment:
This seems totally different to me?

Reply:
So, worth discussion. I set this to 1 because previously the Phase.Threads was 1, so I think only 1 thread was doing something. I could be wrong on that. And maybe 50 was closer to the real intent.